homeassistant/components/heos/media_player.py

```diff
@@ -114,6 +114,8 @@ class HeosMediaPlayer(MediaPlayerEntity):
 
     _attr_media_content_type = MediaType.MUSIC
     _attr_should_poll = False
+    _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
`_attr_name` is never set. Does something like `self._attr_name = player.name` need to be added to `__init__`?

Reply:
https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations. If `_attr_name` is set to None, the device name will be used.

Reply:
Thanks, got it. I'm not up to speed on all the latest changes. 😅

Reply:
Haha, don't worry :)